File: Gamestats/ClientApp/src/app/models/participant.ts

import { Player } from './player';
import { Round } from './round';

export class Participant implements IParticipant {
  participantId: number;
  player: Player;
  round: Round;
  winner: boolean;

  public participant(player: Player) {
    this.player = player;
    this.winner = false;
  }
}

export interface IParticipant {
  player: Player;
  round: Round;
  winner: boolean;
}

File: Gamestats/Models/Participant.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Gamestats.Models
{
    public class Participant
    {
        [ForeignKey("round_id")]
        [Column(name: "round_id")]
        public int RoundId { get; set; }

        public Round Round { get; set; }

        [ForeignKey("player_id")]
        [Column(name: "player_id")]
        public Player Player { get; set; }

        [Key]
        [Column(name: "participant_id")]
        public int ParticipantId { get; set; }

        public bool Winner { get; set; }
    }
}
